### DISCUSSION TITLE
舞蹈动作拆分需求

### DISCUSSION THREAD

**Discussion by geekroscom** [MiRoboticsLab/cyberdog_ros2#12] [Ideas]
可以把目前的舞蹈动作拆分成多个，方便后续”云舞“活动使用，通过远程多机联动配合音乐操控不同设备的不同动作，帮忙尽快安排，提供proto方便后续移植到Python运行。感谢。

**Comment by homalozoa** [comment]
赞同，该功能已经列入计划

**Reply by liseri** [reply]
舞蹈动作本身就是可分的；打滚功能能不能放出toml来啊；

**Reply by homalozoa** [reply]
打滚那个是写死的，放开底层得改些东西
